samples/bluetooth: Add note about Long term key clearance

erwango opened the pull request:
It could happen that bluetooth samples fail because of stored LTK
on device used to test the sample (eg smartphone).

Add a note in bluetooth sample doc to inform users of this trick.

Fixes #13034

Signed-off-by: Erwan Gouriou <erwan.gouriou@linaro.org>

dbkinder commented on samples/bluetooth/bluetooth.rst:
```diff
@@ -16,6 +16,13 @@ then be used with any external Host (including Zephyr running natively or
 documentation and are prefixed with :literal:`hci_` in their folder names.
 
+.. note::
+   Over testing iterations, it might happen Long Term Key (mutually shared
```
How about this clarity edit:
```
   The mutually-shared encryption key created during host-device paring may get
   old after many test iterations.  If this happens, subsequent host-device
   connections will fail. You can force a re-paring and new key to be created
   by removing the device from the associated devices list on the host.
```

erwango replied:
You're the boss. Fixed!

erwango committed:
samples/bluetooth: Add note about Long term key clearance

It could happen that bluetooth samples fail because of stored LTK
on device used to test the sample (eg smartphone).

Add a note in bluetooth sample doc to inform users of this trick.

Fixes #13034

Signed-off-by: Erwan Gouriou <erwan.gouriou@linaro.org>

dbkinder reviewed (approved):
+1 for docs, thanks.
